test/images/scripts/fedora-26.setup

```diff
@@ -89,17 +89,20 @@ dnf $DNF_OPTS -y install $TEST_PACKAGES $COCKPIT_DEPS $IPA_CLIENT_PACKAGES
 dnf $DNF_OPTS -y install mock dnf-plugins-core rpm-build
 useradd -c Builder -G mock builder
 
-# HACK - mock --installdeps with yum is broken, it seems that it can't
-# run any package scriptlets.  Yum is deprecated anyway so we just use
-# dnf.  I wonder why dnf isn't the default anyway...
+
+srpm=$(/var/lib/testvm/make-srpm $TEST_SOURCE)
+
+# HACK - mock --installdeps is broken, it seems that it forgets to
+# copy the source rpm to a location that dnf can actually access.
 #
-echo "config_opts['package_manager'] = 'dnf'" >>/etc/mock/site-defaults.cfg
+# https://bugzilla.redhat.com/show_bug.cgi?id=1447627
 
-# HACK - Switch off gpgcheck while the repos and mirrors settle down
-sed -i -e 's/gpgcheck=1/gpgcheck=0/' /etc/mock/default.cfg
+su builder -c "/usr/bin/mock --init"
+root=/var/lib/mock/fedora-26-x86_64-bootstrap/root
```

Comment:
We will also use this script on fedora-i386, where `x86_64` is wrong. Can we just use a glob here instead?

However, hopefully this will go away soon once mock is fixed (thanks for filing the bug!) - so if a glob doesn't work and we can get rid of this before we move fedora-i386 to fedora 26, it's okay.